integration/ui/uitest/run.go

```diff
@@ -0,0 +1,92 @@
+// Copyright (C) 2021 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package uitest
+
+import (
+	"fmt"
+	"os"
+	"testing"
+	"time"
+
+	"github.com/go-rod/rod"
+	"github.com/go-rod/rod/lib/launcher"
+	"github.com/go-rod/rod/lib/utils"
+	"github.com/stretchr/testify/require"
+	"go.uber.org/zap"
+	"go.uber.org/zap/zaptest"
+
+	"storj.io/common/testcontext"
+	"storj.io/storj/private/testplanet"
+	"storj.io/storj/satellite"
+)
+
+// Test defines common services for uitests.
+type Test func(t *testing.T, ctx *testcontext.Context, planet *testplanet.Planet, browser *rod.Browser)
+
+type zapWriter struct {
+	*zap.Logger
+}
+
+func (log zapWriter) Write(data []byte) (int, error) {
+	log.Logger.Info(string(data))
+	return len(data), nil
+}
+
+// Run starts a new UI test.
+func Run(t *testing.T, test Test) {
+	if os.Getenv("STORJ_TEST_SATELLITE_WEB") == "" {
+		t.Skip("Enable UI tests by setting STORJ_TEST_SATELLITE_WEB to built npm")
+	}
+	if os.Getenv("STORJ_TEST_SATELLITE_WEB") == "omit" {
+		return
```

Comment:
won't this mean that test will be in results as successful test?

Reply:
Unfortunately, yes... there's no nice way to skip particular package from tests without extra effort. So the easier way is to specify "omit" for them.

But, I guess, it would be better to have them listed as skipped.